### DISCUSSION TITLE
Prime INFORMATION Rev. 7.0.4.R2-20.2 Manuals?

### DISCUSSION THREAD

**Discussion by SirNibble** [prirun/p50em#8] [General]
One of the reasons I'm interested in re-living my PR1ME memories is using INFORMATION again. I remember it was quite a fun envirnoment to use.

I've a copy of "Prime info Primer - DOC3909-052 for Rev 5.3" but hoped someone had  manuals, INFO/BASIC in particular, more appropiate for Rev 7.

**Comment by drboone** [comment]
I thought I had an INFO/BASIC manual or pocket ref somewhere, but can't lay hands on it this minute.  I do have an apparently un-scanned FDR5212 "INFORMATION Highlights" that matches v4.5.

Here's what I know of in scanned form:

https://yagi.h-net.org/prime_manuals/pdr3905_inform_refgde_1981.pdf
https://yagi.h-net.org/prime_manuals/pdr3906_perform_refgde_1980.pdf
https://yagi.h-net.org/prime_manuals/prirun_scans/Prime%20Information%20Update%205.3.2%201983.pdf

The 21.0.6 dist tape set https://sysovl.info/pages/blobs/prime/prime/primos_21.0.tar.gz includes INFORMATION.

I know a guy who probably has some paper, though I don't know what version(s).  I should reach out and see if he'd loan them for scanning.

**Comment by SirNibble** [comment]
I've a small stack of Rev19 & 20 Pocket guides, but damned if I can find them now I can use them again!

Thanks for the PDFs. Certainly a help.

**Comment by drboone** [comment]
I got the Highlights pocket guide scanned:

https://sysovl.info/pages/blobs/prime/programmercompanion/fdr5212_infohighlights.pdf

**Comment by SirNibble** [comment]
Got to love

"As long as you
correct errors before hitting the RETURN key, you won't
get besieged by strange error messages. Just keep trying
until you get it right."

They don't write manuals like that anymore!

**Comment by drboone** [comment]
Then there's the "New User's Guide to Editor and Runoff", and the Circular Reasoning cycle in the Fortran manual index. :)

**Comment by MarkHaroldson** [comment]
Wait! Can I really run prime information on this package? Dang, I've got a closet full of Reel-to-Reel tapes from my first job. 

You are correct PI was a fun environment to work in.

**Comment by drboone** [comment]
Prime software, or first-job software/data?  If the former, we should get them read!

Yes, you can run PI.  I think the public rev 21 emulator has it install.

**Reply by MarkHaroldson** [reply]
It was my First Job, I started running the backup tapes at night and quickly became a Info/Basic Programmer/Analyst, 
super cool environment for its time. Now if I can only find a 750 to restore!
